samples/NativeAOT/README.md

```diff
@@ -0,0 +1,89 @@
+# NativeAOT Sample
+
+This is the sample for using NativeAOT with .net Android
+
+## Debugging
+
+In order to debug we need to use `lldb`. First install the application
+
+```dotnetcli
+./dotnet-local.sh build samples/NativeAOT/NativeAOT.csproj -c Debug -t:Install
+```
+
+Then user the following commands to install the `lldb-server` for the application.
+
+```dotnetcli
+adb push $(ANDROID_NDK_HOME)/toolchains/llvm/prebuilt/darwin-x86_64/lib/clang/18/lib/linux/aarch64/lldb-server /data/local/tmp/lldb-server
+adb shell run-as net.dot.hellonativeaot cp /data/local/tmp/lldb-server .
+adb forward tcp:5039 tcp:5039
+adb shell run-as net.dot.hellonativeaot killall -9 lldb-server
+adb shell run-as net.dot.hellonativeaot ./lldb-server platform --listen "*:5039" --server
+```
+
+Note: We have to run the `lldb-server` in the context of the app so it has the correct networking permissions. This is why we use the `run-as` command.
+
+Once `lldb-server` is up and running, you will want to run up the app using the following
+
+```dotnetcli
+adb shell am start -S --user "0" -a "android.intent.action.MAIN" -c "android.intent.category.LAUNCHER" -n "net.dot.hellonativeaot/my.MainActivity" -D
```

Comment:
Should this example use the new `runwithdebugger` scripts in this PR? Wondering if it would make the steps easier.